SuperBuild.cmake

```diff
@@ -4,11 +4,10 @@ slicer_check_cmake_https()
 #-----------------------------------------------------------------------------
 # Git protocol option
 #-----------------------------------------------------------------------------
-option(${CMAKE_PROJECT_NAME}_USE_GIT_PROTOCOL "If behind a firewall turn this off to use http instead." ON)
-
-set(git_protocol "git")
+option(${CMAKE_PROJECT_NAME}_USE_GIT_PROTOCOL "If behind a firewall turn this off to use http instead." OFF)
+set(git_protocol "https")
 if(NOT ${CMAKE_PROJECT_NAME}_USE_GIT_PROTOCOL)
-  set(git_protocol "http")
+  set(git_protocol "git")
```

Comment:
Is this correct - if not "use_git_protocol" then use git?

Reply:
Pushed update to fix my typo.